src/Logic/operate.js

```diff
@@ -0,0 +1,30 @@
+const Big = require('big.js');
```

Comment:
Let's follow the required change committed in the package.json file -> `big-js` not `big.js`